09smart-contracts-security.asciidoc

```diff
@@ -319,6 +319,7 @@ contract TimeLock {
     function withdraw() public {
         require(balances[msg.sender] > 0);
         require(now > lockTime[msg.sender]);
+        balance = balances[msg.sender];
```

Comment:
Need type definition of uint. correctly below:
uint balance = balances[msg.sender];